amqp/src/main/scala/akka/stream/alpakka/amqp/AmqpSinkStage.scala

```diff
@@ -9,22 +9,27 @@ import akka.util.ByteString
 import com.rabbitmq.client.AMQP.BasicProperties
 import com.rabbitmq.client._
 
-final case class OutgoingMessage(bytes: ByteString, immediate: Boolean, mandatory: Boolean, props: Option[BasicProperties])
+final case class OutgoingMessage(bytes: ByteString,
+                                 immediate: Boolean,
+                                 mandatory: Boolean,
+                                 props: Option[BasicProperties])
```

Comment:
@johanandren Probably. If scalafmt can keep the first argument on the same line like the class name, it will do that and align the other arguments, but only if all lines don't get too long. Else it will switch to what you consider "just right".